.mailmap

```diff
@@ -20,6 +20,7 @@ John Regan <john@jrjrtech.com> John Regan <jregan@mesonet.org>
 Jürgen Buchmüller <pullmoll@t-online.de> Juergen Buchmueller <pullmoll@t-online.de>
 Leah Neukirchen <leah@vuxu.org> Christian Neukirchen <chneukirchen@gmail.com>
 Logen Kain <logen@sudotask.com> Logen Kain <walach.of.harkon@gmail.com>
+Matt Boehlke <git@mtboehlke.com> Mat Boehlke <mtboehlke@gmail.com>
```

Comment:
intentionally missing t?

Reply:
Yeah, that's how it was in my old git config

Reply:
ok, just checking